src/components/TechnicalStandards/TechnicalStandards.astro

```diff
@@ -0,0 +1,120 @@
+---
+import { Typography } from "@chainlink/blocks"
+
+const cards = [
+  {
+    title: "Digital Transfer Agent (DTA)",
+    description: "You might be interested in this end-to-end solution. It can solve your problems.",
+    image: "/images/ccip/ccip-hero-bg.png",
+    href: "/",
+  },
+  {
+    title: "Delivery vs. Payment (DvP)",
+    description: "You might be interested in this end-to-end solution. It can solve your problems.",
+    image: "/images/code-sample.png",
+    href: "/",
+  },
+]
+---
+
+<div class="technical-standards">
+  {
+    cards.map((card) => (
+      <a href={card.href} class="standard-card">
+        <div class="card-image" style={`background-image: url(${card.image})`} />
+        <div class="card-body">
+          <div class="card-text">
+            <Typography variant="h5">{card.title}</Typography>
+            <Typography variant="body-l" color="muted">
+              {card.description}
+            </Typography>
+          </div>
+        </div>
+        <img src="/assets/icons/upper-right-arrow.svg" class="card-arrow" />
+      </a>
+    ))
+  }
+</div>
+
+<style>
+  .technical-standards {
+    display: grid;
+    grid-template-columns: 1fr;
+    border: 1px solid var(--border);
+  }
+
+  .standard-card {
+    display: flex;
+    flex-direction: column;
+    padding: var(--space-6x);
+    gap: var(--space-6x);
+    align-items: start;
+  }
+
+  .standard-card:hover {
+    background-color: var(--muted);
+  }
+
+  .standard-card:hover .card-arrow {
+    opacity: 1;
+  }
+
+  .card-image {
+    width: 140px;
+    height: 140px;
+    background-size: 250%;
+    background-position: left;
+    background-repeat: no-repeat;
+  }
+  .standard-card:first-child {
+    border-bottom: 1px solid var(--border);
+  }
+
+  .standard-card:last-child .card-image {
+    background-position: -55px -50px;
+  }
+
+  .standard-card:first-child .card-image {
+    background-position: right;
+  }
+
+  .card-body {
+    display: flex;
+    flex-direction: column;
+    gap: var(--space-4x);
+    flex: 1;
+  }
+
+  .card-text {
+    display: flex;
+    flex-direction: column;
+    gap: var(--space-4x);
+  }
```

Comment:
Merge the two if possible and to avoid future bugs:

```
  .card-body, .card-text  {
    display: flex;
    flex-direction: column;
    gap: var(--space-4x);
  }

  .card-body {
    flex: 1;
  }
```

Reply:
done